bukkit/build.gradle

```diff
@@ -48,6 +52,8 @@ shadowJar {
         exclude(dependency(':gson'))
     }
 
+    mergeServiceFiles()
```

Comment:
This was necessary to properly support the newer version of ebean, since it works with services, by default shadowJar overrides all files instead of merging them.